app/packages/core/src/plugins/SchemaIO/components/NativeModelEvaluationView/evaluation/scenario/Scenarios.tsx

```
@@ -0,0 +1,1439 @@
+  const { scenarios } = evaluation;
+  const [scenario, setScenario] = useState(getDefaultScenario(scenarios));
+  const [selectedSubsets, setSelectedSubsets] = useState(["all"]);
+  const promptOperator = usePanelEvent();
+  const panelId = usePanelId();
+  const [mode, setMode] = useState("charts");
+  const [differenceMode, setDifferenceMode] = useState("percentage");
+  const [loadingScenario, setLoadingScenario] = useState(false);
+  const [loading, setLoading] = useState(false);
+  const evaluationInfo = evaluation.info;
+  const evaluationConfig = evaluationInfo.config;
+  const { key, compareKey, id: eval_id } = data?.view;
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)